sljit_src/sljitLir.h

```diff
@@ -344,7 +344,7 @@ extern "C" {
 #define SLJIT_ARG_TYPE_SCRATCH_REG 0x8
 
 /* Void result, can only be used by SLJIT_ARG_RETURN. */
-#define SLJIT_ARG_TYPE_VOID	0
+#define SLJIT_ARG_TYPE_V	0
```

Comment:
When I try to build by msvc, `SLJIT_ARG_TYPE_void` is used instead of `SLJIT_ARG_TYPE_VOID`.
So, I change the name and it works.
I'm not sure the reason of error